### PR TITLE
Update Argo and Akuity

### PR DESCRIPTION
**Pull request by wojtekidd**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] Are you only including a `repo_url` if your project is 100% open source? If so, you need to pick the single best GitHub repository for your project, not a GitHub organization.
* [x] Is your project closed source or, if it is open source, does your project have at least 300 GitHub stars?
* [x] Have you picked the single best (existing) category for your project?
* [x] Does it follow the other guidelines from the [new entries](https://github.com/cncf/landscape#new-entries) section?
* [x] Have you added your SVG to `hosted_logos` and referenced it there?
* [x] Does your logo clearly state the name of the project/product and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [x] Does your project/product name match the text on the logo?
* [x] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?
* [x] ~30 minutes after opening the pull request, the CNCF-Bot will post the URL for your staging server. Have you confirmed that it looks good to you and then added a comment to the PR saying "LGTM"?

### CODE REVIEW

**Comment by wojtekidd** [comment]
LGTM

File: landscape.yml [hunk 1]
```diff
@@ -4379,7 +4399,7 @@ landscape:
               artwork_url: https://github.com/cncf/artwork/tree/master/projects/argo
               stack_overflow_url: https://stackoverflow.com/questions/tagged/argoproj+or+argo
               blog_url: https://blog.argoproj.io/
-              slack_url: https://argoproj.slack.com/
+              slack_url: https://slack.cncf.io/
```
Comment (terrytangyuan) [inline landscape.yml:4402]: Use this link: https://argoproj.github.io/community/join-slack/

**Comment by wojtekidd** [comment]
LGTM

File: landscape.yml [hunk 2]
```diff
@@ -1975,6 +1975,23 @@ landscape:
               artwork_url: https://github.com/cncf/artwork/blob/master/examples/incubating.md#cilium-logos
               stack_overflow_url: https://stackoverflow.com/questions/tagged/cilium
               slack_url: https://cilium.herokuapp.com/
+              summary_personas: Platform engineers, Network teams, SRE, DevOps Engineers, Security teams
```
Comment (terrytangyuan) [inline landscape.yml:1978]: Is this change intended?

Comment (wojtekidd) [reply landscape.yml:1978]: That's not mine - I merged it with the rest of the changes today when rebasing :)

Comment (terrytangyuan) [reply landscape.yml:1978]: Ok I think you'll want to make sure the diff only contains the changes you made. Probably need to do a git pull --rebase. Perhaps the easiest way for you is to create a brand new fork/branch and send a new PR.
